tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -94,6 +100,16 @@ EventsAndTriggers:
           CoordinatesFloatingPointType: Double
           FloatingPointTypes: [Double]
           OverrideObservationValue: None
+  - - Slabs:
+        EvenlySpaced:
+          Interval: 1000
+          Offset: 0
+    - - Axis1
+  - - Slabs:
+        EvenlySpaced:
+          Interval: 1000
+          Offset: 0
+    - - Axis2
```

Comment:
Add this event right after Axis1 and that trigger since these are the same triggers